Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Archi and kil

## Changelog:
- Added the Warblade of Archimonde artifact.
- Added a new icon for the Scepter of Sargeras.
- Kil'jaeden also gets the Legion Doomhorn when spawned.
- Kil'jaeden now gets the Scepter of Sargeras as he owned it according to the lore.
- Added new portraits for Archimonde and Kil'jaeden.
- Archimonde is now lvl 8 warlock and lvl 1 warrior.

# How to test:
Use `event WCFEL.1000` console command, check if Archimonde has a portrait and his warblade, do the same for Kil'jaeden but check for the scepter instead.
